src/components/CreateProfile/CreateProfileTopBar/CreateProfileTopBar.tsx

```diff
@@ -3,13 +3,16 @@ import styles from './CreateProfileTopBar.module.scss'
 import { Button } from '@/inputs/Button/Button'
 import { ErrorIcon } from '../../../assets/icons/ErrorIcon'
 import { useFormikContext } from 'formik'
+import { useRouter } from 'next/navigation'
 
 const ProfileTopBar = () => {
+  const router = useRouter()
   const { handleSubmit, errors } = useFormikContext()
 
   const handleButtonClick = (event: React.MouseEvent<HTMLButtonElement>) => {
     event.preventDefault()
     handleSubmit()
+    router.push('/my-profile')
```

Comment:
**nitpick:** We should create some dictionary with available routes in our app. It will be hard to maintain it otherwise. I mean eg, create `utils/routes.ts` and place there:

```ts
export enum AppRoutes {
 myProfile = '/my-profile',
 ...
}
```